.env_example

```diff
@@ -10,6 +10,4 @@ export APP_ENV=local
 
 export PORT=3000
 
-export SDR_USER=sdrUser
 export SDR_PASS=sdrPass
-
```

Comment:
The reason that ENV values were used in these configs is because, at the time, we did not have a policy around security for login information in deploy files.  I raised the question about security of these login details and assumed it safer to avoid public publication of these details and, further, assumed they could be ENV values set by the puppet code.  All of these issues have since been clarified, so I doubt we need this file and most of the ENV settings in these configs.  If possible, I'd like this file removed and, possibly, the use of the env gem and other associated ENV values, in favour of using the rails-config gem to set Settings.